rcl_yaml_param_parser/include/rcl_yaml_param_parser/parser.h

```diff
@@ -57,9 +57,11 @@ RCL_YAML_PARAM_PARSER_PUBLIC
 void rcl_yaml_node_struct_fini(
   rcl_params_t * params_st);
 
-/// \brief Parse the YAML file, initialize and populate params_st
+/// \brief Parse the YAML file and populate \p params_st
+/// \pre Given \p params_st must be a valid parameter struct
+///   as retured by `rcl_yaml_node_struct_init()`
```

Comment:
```suggestion
///   as returned by `rcl_yaml_node_struct_init()`
```

Reply:
Argh... :facepalm: See https://github.com/ros2/rcl/pull/776/commits/be78d0d955ec55aef2493eb9b9f6dd0d9ee3d047. Wonder why my spell checker didn't catch it.